CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ target_include_directories(
 target_link_libraries(
     ice-engine-sdk 
     PUBLIC
+    sndio
```

Comment:
Thanks for adding this. We will have to figure out if CMake has a straightforward way to add it to only some variants of Linux.